### PR TITLE
🚀 Remove custom-elements and IntersectionObserver polyfill for SxG.

### PR DESCRIPTION
**Pull request by xiexr151e**
@jridgewell mentioned that the custom-elements polyfill could be removed for the SxG build. This PR is a one line change that achieves that.
In addition, the IntersectionObserver polyfill can also be removed as mentioned by @erwinmombay.

### CODE REVIEW

**Comment by xiexr151e** [comment]
@jridgewell Kris mentioned that IntersectionObserver may also not be needed in SxG, should I just remove that in this PR also?

**Comment by xiexr151e** [comment]
> We also need to strip in in `src/friendly-iframe-embed-legacy.js` and `src/friendly-iframe-embed.js`

Just finished.

**Comment by kevinkimball** [comment]
@xiexr151e  What is the size reduction?

**Comment by xiexr151e** [comment]
> @xiexr151e What is the size reduction?

Removing both polyfills shaved off approximately 2.2KB. Will give exact numbers in a bit.

File: src/friendly-iframe-embed-legacy.js [hunk 1]
```diff
@@ -280,6 +280,8 @@ function installPolyfillsInChildWindow(parentWin, childWin) {
   installDOMTokenList(childWin);
```
Comment (jridgewell) [inline src/friendly-iframe-embed-legacy.js:280]: Actually, the above two aren't needed for ESM, either.

File: src/friendly-iframe-embed.js [hunk 2]
```diff
@@ -758,8 +758,10 @@ function installPolyfillsInChildWindow(parentWin, childWin) {
   installDOMTokenList(childWin);
```
Comment (jridgewell) [inline src/friendly-iframe-embed.js:758]: Ditto.

**Pull request merged**